File: WebApplication1/WebApplication1/DTO/Response/UserResDto.cs
using WebApplication1.Models;

namespace WebApplication1.DTO.Response
{
    public class UserResDto
    {
        public int UserId { get; set; }
        public string UserName { get; set; } = null!;
       public required string Email { get; set; }
        public UserRole Role { get; set; } = UserRole.AUDIENCE;
    }
}

File: angular-app/src/app/api/models/user-res-dto.ts
/* tslint:disable */
/* eslint-disable */
import { BookingResDto } from './booking-res-dto';
import { ConcertResDto } from './concert-res-dto';
import { UserRole } from './user-role';
export interface UserResDto {
  bookings?: null | Array<BookingResDto>;
  concerts?: null | Array<ConcertResDto>;
  email: null | string;
  role?: UserRole;
  userId?: number;
  userName?: null | string;
}
